apps/src/blockTooltips/DropletTooltipManager.js

```diff
@@ -124,7 +126,7 @@ DropletTooltipManager.prototype.showDocFor = function (functionName) {
   }
 
   $('.tooltipstered').tooltipster('hide');
-  var dialog = new window.Dialog({
+  var dialog = this.Dialog({
```

Comment:
Why did we lose the `new` keyword here?  If intentional, why is this named `Dialog` instead of `dialog`?